src/applications/vaos/actions/newAppointment.js

```diff
@@ -361,7 +383,9 @@ export function submitAppointmentOrRequest(router) {
         let requestBody;
         let requestData;
 
-        if (newAppointment.data.facilityType === 'communityCare') {
+        if (
+          newAppointment.data.facilityType === FACILITY_TYPES.COMMUNITY_CARE
```

Comment:
Checking facility types is being used frequently throughout the app, so I thought it'd be a good idea to create a constant for them.